src/LayerUtil/LayerUtil.ts

```diff
@@ -38,11 +38,14 @@ class LayerUtil {
   ): string => {
     const layerSource = layer.getSource();
 
-    if (layerSource instanceof OlSourceTileWMS) {
+    if (layerSource?.constructor?.name ===  OlSourceTileWMS.name) {
+      // @ts-ignore
```

Comment:
I don't know whtas wrong here but you may want to adjust this in the tsconfig instead.

Reply:
replaced by typeguards